### PR TITLE
[NCL-4392] Expose info about num of building and enqued builds

### PR DESCRIPTION
**Pull request by thescouser89**
### Checklist:

* [ ] Have you added a note in the CHANGELOG.md for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by thescouser89** [comment]
retest this please

**Comment by thescouser89** [comment]
I won't add a changelog since it'll be in the next 1.4.x release.

File: rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java [hunk 1]
```diff
@@ -196,6 +197,32 @@ public BuildRecordProvider(
...
+        List<BuildTask> x = buildCoordinator.getSubmittedBuildTasks();
+
+        int waitingForDependencies = 0;
+        int running = 0;
+        int enqueued = 0;
+
+        for (BuildTask task: x) {
+            switch(task.getStatus()) {
+                case ENQUEUED:
+                    enqueued += 1;
```
Comment (jbartece) [inline rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java]: Don't be afraid to use enqueued++; ;-)

Comment (thescouser89) [reply rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java]: Oh woops you're right! let me know if you want me to change it!

Comment (thescouser89) [reply rest/src/main/java/org/jboss/pnc/rest/provider/BuildRecordProvider.java]: done!

**Pull request merged**